### PR TITLE
Add Portuguese Municipalities

### PR DESCRIPTION
**Pull request by dreispt**
Depends on https://github.com/OCA/partner-contact/pull/270

### CODE REVIEW

**Comment by dreispt** [comment]
@andreparames can you make a code review please?

**Comment by dreispt** [comment]
@andreparames can you also review https://github.com/OCA/partner-contact/pull/270 ? :construction_worker:

File: l10n_pt_municipality/README.rst [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+=============
+
+The setup tables are available under the menu:
+Sales > Configuration > Address Book
+
+
+Usage
+=====
+
+Onle the Administrator user has acces to the LAU setup table.
```
Comment (andreparames) [inline l10n_pt_municipality/README.rst]: Just a couple of typos.

File: l10n_pt_municipality/README.rst [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
+Contributors
+------------
+
+* Daniel Reis (dreis.pt at hotmail.com)
```
Comment (andreparames) [inline l10n_pt_municipality/README.rst:67]: This is not a bug, but I noticed it's different from OCA/partner-contact#270

**Comment by andreparames** [comment]
LGTM :+1:

**Comment by dreispt** [comment]
Fixed.

**Comment by dreispt** [comment]
Waiting on https://github.com/OCA/partner-contact/pull/270...

**Comment by dreispt** [comment]
TravisCi is green, merging.

**Pull request merged**